src/features/Login/PasswordForgotConfirmation.tsx

```diff
@@ -1,48 +1,90 @@
 import { Box, Flex } from '@rebass/grid';
 import * as React from 'react';
 import Helmet from 'react-helmet';
-import { Link } from 'react-router-dom';
-
-import resetLogo from '../../assets/images/passwordReset.svg';
+import MediaQuery from 'react-responsive';
+import { BrowserRouter, Link } from 'react-router-dom';
 
 import { FrontendRoute } from '../../config';
 import {
-  Button,
+  BackgroundImage,
   H1,
-  Image,
+  LeftContainer,
   MaxWidthBox,
   Paragraph,
 } from '../../shared/Elements';
+import Button, { ButtonVariant } from '../../shared/Elements/Button';
+
+import BackgroundLandscape from '../../assets/images/backgroundLandscape.svg';
 
 class PasswordResetEmailConfirmationContainer extends React.Component<{}, {}> {
   public render() {
+    return (
+      <MediaQuery minWidth={1224}>
+        {(matches) =>
+          matches ? (
+            <LeftContainer>
+              {this.renderPassForgotConfirmation()}
+              <BackgroundImage
+                src={BackgroundLandscape}
+                top={'0px'}
+                left={'0px'}
+                imgWidth={'100%'}
+                imgHeight={'100%'}
+                minHeight={'600px'}
+              />
+            </LeftContainer>
+          ) : (
+            <div>
+              {this.renderPassForgotConfirmation()}
+              <BackgroundImage
+                src={BackgroundLandscape}
+                top={'0px'}
+                left={'0px'}
+                imgHeight={'100%'}
+              />
+            </div>
+          )
+        }
+      </MediaQuery>
+    );
+  }
+
+  private renderPassForgotConfirmation() {
     return (
       <Flex
         justifyContent={'center'}
-        alignItems={'center'}
+        alignItems={'left'}
         flexDirection={'column'}
+        p={'5rem 0rem 0rem 6.4rem'}
       >
         <Helmet>
-          <title>Password Sent | McHacks 6</title>
+          <title>Password Sent | McHacks 7</title>
         </Helmet>
         <Flex alignItems={'center'}>
           <Box>
-            <Image src={resetLogo} imgHeight={'3rem'} padding={'1.0rem'} />
-          </Box>
-          <Box>
-            <H1>Password reset</H1>
+            <H1 fontSize={'24px'} marginLeft={'0px'}>
+              Reset Your Password
+            </H1>
           </Box>
         </Flex>
-        <MaxWidthBox width={1} fontSize={[2, 3, 4]}>
-          <Paragraph paddingBottom={'20px'} textAlign={'center'}>
+        <MaxWidthBox width={'60%'} fontSize={[2, 3, 4]}>
+          <Paragraph
+            paddingBottom={'20px'}
+            textAlign={'left'}
+            fontSize={'18px'}
+          >
             We've sent you a link to reset your password. Check your inbox and
             follow the instructions there.
           </Paragraph>
         </MaxWidthBox>
         <Box>
-          <Link to={FrontendRoute.LOGIN_PAGE}>
-            <Button type="button">Login page</Button>
-          </Link>
+          <BrowserRouter forceRefresh={true}>
```

Comment:
What does BrowserRouter do here?

Reply:
since the forgot password email request confirmation page is at the same path as the actual request page, and i wanted the button on the confirmation page to bring back to the request page, i just had it force refresh when it was clicked. This will be changed later, just a hacky way to do it for now